docs/examples/host-guest-implicit.rst

```diff
@@ -113,12 +93,55 @@ In this case, the ``OBC2`` name is the Onufriev-Bashford-Case GBSA model. We cou
 defaults to the dielectric for TIP3P water.
 
 
-Reamining Headings
-^^^^^^^^^^^^^^^^^^
+Systems and Protocols Headings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The remaining headings: ``systems``, ``protocols``, and ``experiments`` are straightforward. Please see the more
+The headings ``systems`` and ``protocols`` are straightforward. Please see the more
 :doc:`detailed p-xylene in T4 Lysozyme example <p-xylene-explicit>` for more information on these sections.
 
+Note however in the ``protocols`` heading there is not a ``lambda_restraints`` specified, the reason for that is the
+type of restraint we choose as we discuss below.
+
+Experiments Heading
+^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: yaml
+
+    experiments:
+      system: host-guest
+      protocol: absolute-binding
+      restraint:
+        type: FlatBottom
+
+This ``experiments`` heading differs from the :doc:`p-xylene binding example <p-xylene-explicit>` due to the type of
+restraint we chose, in this case, ``FlatBottom``.
+
+We apply a ``FlatBottom`` restraint
+to keep the guest from wondering too far away from the host. The ``FlatBottom`` option applies no biasing potential
+until the guest drifts too far away at which point a harmonic bias is applied relative to the geometric center of the
+host. It loosely follows this equation:
+
+.. math::
+
+
+  U_{\text{restraint}}(r) =  \begin{cases}
+                                               0 & \,\text{if}\, r \le r_{\text{cutoff}} \\
+      K \cdot \left(r-r_{\text{cutoff}} \right)^2 & \,\text{if}\, r > r_{\text{cutoff}}
+      \end{cases}
+
+The Flat Bottom restraint has another advantage on this system in particular. The symmetric host molecule is known to
+have multiple binding sites that are hard to locate in normal simulation time lengths. The ``FlatBottom`` restraint allows
+the guest to explore more of the space around the host relative to the other restraint type ``Harmonic``.
+
+The ``Harmonic``
+restraint type would keep the ligand close to **a** binding site, but not both. It should be noted though, that if you
```

Comment:
It may be worth specifying that it keeps it close to the centroid of the receptor at the moment, so that the user doesn't expect our code to automatically detect binding sites.